backend/de.metas.business/src/main/sql/postgresql/system/20-de.metas.business/5697590_sys_gh15938_Mark_As_Fully_Delivered_2.sql

```diff
@@ -0,0 +1,159 @@
+;
+
+-- Element: NotFullyDelivered
+-- 2023-08-01T09:01:51.894Z
+UPDATE AD_Element_Trl SET Name='Nicht vollständig ausgeliefert', PrintName='Nicht vollständig ausgeliefert',Updated=TO_TIMESTAMP('2023-08-01 12:01:51','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Element_ID=582619 AND AD_Language='de_CH'
+;
+
+-- 2023-08-01T09:01:51.925Z
+/* DDL */  select update_TRL_Tables_On_AD_Element_TRL_Update(582619,'de_CH')
+;
+
+-- Element: NotFullyDelivered
+-- 2023-08-01T09:01:55.357Z
+UPDATE AD_Element_Trl SET Name='Nicht vollständig ausgeliefert',Updated=TO_TIMESTAMP('2023-08-01 12:01:55','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Element_ID=582619 AND AD_Language='de_DE'
+;
+
+-- 2023-08-01T09:01:55.359Z
+/* DDL */  select update_TRL_Tables_On_AD_Element_TRL_Update(582619,'de_DE')
+;
+
+-- Element: NotFullyDelivered
+-- 2023-08-01T09:02:03.690Z
+UPDATE AD_Element_Trl SET PrintName='Nicht vollständig ausgeliefert',Updated=TO_TIMESTAMP('2023-08-01 12:02:03','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Element_ID=582619 AND AD_Language='de_DE'
+;
+
+-- 2023-08-01T09:02:03.706Z
+/* DDL */  select update_TRL_Tables_On_AD_Element_TRL_Update(582619,'de_DE')
+;
+
+-- Column: C_Order.NotFullyDelivered
+-- 2023-08-01T09:03:05.108Z
+INSERT INTO AD_Column (AD_Client_ID,AD_Column_ID,AD_Element_ID,AD_Org_ID,AD_Reference_ID,AD_Table_ID,CloningStrategy,ColumnName,ColumnSQL,Created,CreatedBy,DDL_NoForeignKey,DefaultValue,EntityType,FacetFilterSeqNo,FieldLength,IsActive,IsAdvancedText,IsAllowLogging,IsAlwaysUpdateable,IsAutoApplyValidationRule,IsAutocomplete,IsCalculated,IsDimension,IsDLMPartitionBoundary,IsEncrypted,IsExcludeFromZoomTargets,IsFacetFilter,IsForceIncludeInGeneratedModel,IsGenericZoomKeyColumn,IsGenericZoomOrigin,IsIdentifier,IsKey,IsLazyLoading,IsMandatory,IsParent,IsRestAPICustomColumn,IsSelectionColumn,IsShowFilterIncrementButtons,IsShowFilterInline,IsStaleable,IsSyncDatabase,IsTranslated,IsUpdateable,IsUseDocSequence,MaxFacetsToFetch,Name,SelectionColumnSeqNo,SeqNo,Updated,UpdatedBy,Version) VALUES (0,587241,582619,0,20,259,'XX','NotFullyDelivered','( CASE           WHEN (SELECT SUM(qtydelivered - qtyordered) from C_OrderLine where C_Order_ID = C_Order.c_order_id) >= 0 THEN ''N''    ELSE ''Y''   END )',TO_TIMESTAMP('2023-08-01 12:03:04','YYYY-MM-DD HH24:MI:SS'),100,'N','Y','D',0,1,'Y','N','Y','N','N','N','N','N','N','N','Y','N','N','N','N','N','N','Y','N','N','N','N','N','N','N','N','N','N','N',0,'Not Fully Delivered',0,0,TO_TIMESTAMP('2023-08-01 12:03:04','YYYY-MM-DD HH24:MI:SS'),100,0)
```

Comment:
> ( CASE WHEN (SELECT SUM(qtydelivered - qtyordered) from C_OrderLine where C_Order_ID = C_Order.c_order_id) >= 0 THEN ''N'' ELSE ''Y'' END )'

not sure if this will work right for draft orders with lines having qtyOrdered 0

Reply:
@cp-ps I added a fix